ark/analysis/spatial_analysis.py

```diff
@@ -133,13 +132,13 @@ def calculate_cluster_spatial_enrichment(all_data, dist_matrices_dict, included_
         dist_lim (int):
             cell proximity threshold. Default is 100
         fov_col (str):
-            column with the cell fovs. Default is 'SampleID'
+            column with the cell fovs. Default is eqaul to `ark.settings.FOV_ID`
```

Comment:
Same thing. I don't think there's any benefit to explaining what the default is when the default is in the arg list